index.html

```diff
@@ -3432,7 +3432,7 @@ <h5>JWT Encoding</h5>
 
             <ul>
               <li>
-<code>alg</code> MUST be used for RSA and ECDSA-based digital signatures. If
+<code>alg</code> MUST be used for digital signatures. If
 only the <code>proof</code> attribute is used, the <code>alg</code> header MUST
```

Comment:
incorporating #671 here:

```suggestion
only the <code>proof</code> property is used, the <code>alg</code> header MUST
```